backuptar/tar.go

```diff
@@ -317,32 +319,34 @@ func FileInfoFromHeader(hdr *tar.Header) (name string, size int64, fileInfo *win
 // tar file that was not processed, or io.EOF is there are no more.
 func WriteBackupStreamFromTarFile(w io.Writer, t *tar.Reader, hdr *tar.Header) (*tar.Header, error) {
```

Comment:
warning: cyclomatic complexity 28 of function WriteBackupStreamFromTarFile() is high (> 15) (gocyclo)